content/ledger-objects/ripplestate.md

```diff
@@ -0,0 +1,102 @@
+## RippleState
+[[Source]<br>](https://github.com/ripple/rippled/blob/5d2d88209f1732a0f8d592012094e345cbe3e675/src/ripple/protocol/impl/LedgerFormats.cpp#L70 "Source")
+
+The `RippleState` object type connects two accounts in a single currency. Conceptually, a `RippleState` object represents two _trust lines_ between the accounts, one from each side. Each account can change the settings for its side of the `RippleState` object, but the balance is a single shared value. A trust line that is entirely in its default state is considered the same as trust line that does not exist, so `rippled` deletes `RippleState` objects when their properties are entirely default.
```

Comment:
Sorry, not adding much more value than copyedits... but here's another tiny one: change "considered the same as trust line that" to:

> considered the same as a trust line that